components/_util/wave.tsx

```diff
@@ -98,6 +98,11 @@ export default class Wave extends React.Component<{ insertExtraNode?: boolean }>
         'antd-wave',
         { csp: this.csp },
```

Comment:
this could use `attachTo` option instead of below appendChild maybe? (adopted the below part from an older master commit)